utils/utils.go

```diff
@@ -114,3 +116,10 @@ func IsDebugEnabled() (bool, string) {
 		return true, debugValue
 	}
 }
+
+func HandleNoResults(ios *iostreams.IOStreams, msg string) error {
+	if ios.CanPrompt() {
```

Comment:
I think `ios.IsStdoutTTY() && ios.IsStderrTTY()` is more appropriate here. `ios.CanPrompt()` also checks stdin which I don't think is necessary.

Reply:
This makes sense. I sorta blindly followed what we were doing [here](https://github.com/cli/cli/pull/5479/files#diff-41e81b837f1b925b03131c1133e60c5c52f16b8addb99b10fbd6a5fcc01509bfL45-L46), but there's no reason to continue doing that

Reply:
Actually I'm unconvinced -- if I run:
```bash
gh pr list
```
and see an `stderr` message `"There are no open pull requests in user/repo"`, then part of me would expect to be able to do:
```bash
gh pr list 2> err.log
```
and properly see the same `stderr` message. Maybe we should only be checking `io.IsStdoutTTY()`?

I'll push this approach in my update.

Reply:
Agreed with leaving it as just `io.IsStdoutTTY()`.